internal/server/server.go

```diff
@@ -306,7 +306,7 @@ func handleScheduleRuleState(now time.Time, r *api.Rule, state string) error {
 	if toStart {
 		return startRule(r.Id)
 	} else if toStop {
-		stopRule(r.Id)
+		stopRule(r.Id, true)
```

Comment:
schedule start/stop does not need to update trigger status in the db. Can we call a new function instead of changing the existing function?